packages/compass-import-export/src/stores/export-store.ts

```diff
@@ -1,54 +1,66 @@
 import type AppRegistry from 'hadron-app-registry';
-import { createStore, applyMiddleware } from 'redux';
-import type { Store, AnyAction } from 'redux';
-import thunk from 'redux-thunk';
+import { configureStore } from '@reduxjs/toolkit';
```

Comment:
I was chatting with @mcasimir on how we can do this store and reducers. We agreed going for consistency in the app is best. Although using [`@reduxjs/toolkit`](https://redux-toolkit.js.org/usage/usage-guide) is [recommended](https://react-redux.js.org/using-react-redux/usage-with-typescript#standard-redux-toolkit-project-setup-with-typescript), we aren't using it in other places in Compass. So I'll remove this and go back to how we do it in other places in Compass. This will involve removing the `createSlice` in the reducer and doing things similar to how we're using redux elsewhere.
`@reduxjs/toolkit` docs: https://redux-toolkit.js.org/usage/usage-guide